src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -227,6 +240,7 @@ private void sendConsumerRecordsResponse(HttpServerResponse response, Buffer buf
     }
 
     private void sendConsumerDeletionResponse(HttpServerResponse response) {
+        HttpSinkBridgeEndpoint.consumerIds.remove(this.consumerInstanceId);
```

Comment:
I have a list of consumer IDs. When we want to create a consumer, we check whether the user with this ID is already present or not. So when we delete consumer, we have to remove it from this list. Otherwise, it would not possible to recreate this consumer.

Reply:
I see but the consumer ids are handled by the upper level HttpBridge, we don't have to duplicate the logic in the endpoint with another list. I got your point but I also changed the code so I guess that it's better closing this PR without merging it. Do you agree?

Reply:
Yeah.